tests/test_security_analysis.py

```diff
@@ -260,6 +260,5 @@ def test_different_equipment_contingency():
     sa.add_single_element_contingency('TWT', 'Twt contingency')
     sa.add_single_element_contingency('S1VL1_BBS1_GEN_DISCONNECTOR', 'Switch contingency')
     sa_result = sa.run_ac(n)
-    assert 'Load contingency' in sa_result.post_contingency_results.keys()
```

Comment:
something seems to be wrong

Reply:
the load contingency is disappearing juste after the run of the Runner (OLF)

Reply:
here : https://github.com/powsybl/pypowsybl/blob/main/java/src/main/java/com/powsybl/python/security/SecurityAnalysisContext.java#L31

Reply:
@EtienneLt we have a bug... https://github.com/powsybl/powsybl-open-loadflow/pull/768